src/pages/transactions/_components/TransactionRow.tsx

```diff
@@ -0,0 +1,195 @@
+              <TransactionLinkRow
+                label="From:"
+                pathname={`/address/${data.hash}`}
+                value={data.from}
+                containerClass="flex flex-col lg:flex-row col-start-1 col-end-3"
+              />
+              <TransactionLinkRow
+                label="To:"
+                pathname={`/address/${data.hash}`}
+                value={data.from}
+                containerClass="flex flex-col lg:flex-row col-start-3 col-end-5"
+              />
+            </div>
+          </div>
+          <div className="col-start-7 col-end-9 lg:col-start-11 lg:col-end-13 text-white-50 justify-self-end">
+            <StatusComponent status={data.status} />
+            <div className="text-right mr-8 mt-2 lg:mt-3">
+              <TimeComponent time={data.time} />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* For mobile */}
+      <div className="md:hidden py-6 z-10 mt-2">
+        <div className="flex flex-row justify-between">
+          <div className="flex flex-row">
+            <Icon size={24} className="text-white-50 stroke-white-50" />
+            <div className="flex flex-col overflow-hidden ml-2 text-base">
+              <span className="text-white-50">{data.transactionType}</span>
+            </div>
+          </div>
+          <div className="text-right">
+            <StatusComponent status={data.status} />
+          </div>
+        </div>
+        <div className="ml-8">
+          <AmountComponent
+            amount={data.amount}
+            symbol={data.symbol}
+            containerClass="mt-4"
+          />
+          <TimeComponent time={data.time} containerClass="mt-4" />
+          <TransactionLinkRow
+            label="Hash:"
+            pathname={`/transactions/${data.hash}`}
+            value={data.hash}
+            containerClass="flex flex-row mt-4 w-5/6"
+          />
+          <TransactionLinkRow
+            label="From:"
+            pathname={`/address/${data.from}`}
+            value={data.hash}
+            containerClass="flex flex-row mt-4 w-5/6"
+          />
+          <TransactionLinkRow
+            label="To:"
+            pathname={`/address/${data.to}`}
+            value={data.hash}
+            containerClass="flex flex-row mt-4 w-5/6"
+          />
+        </div>
+      </div>
+      <div className="border-b border-black-600 ml-8 md:ml-10" />
+    </div>
+  );
+}
+
+function StatusComponent({ status }: { status: string }): JSX.Element {
+  const Icon =
+    status === TransactionStatus.Confirmed ? ConfirmCheck : RejectedCross;
+  return (
+    <div className="flex flex-row">
+      <span className="text-white-700 text-base hidden lg:block">Status:</span>
+      <span
+        className={clsx(
+          "hidden md:block text-base lg:ml-1 mr-2",
+          status === TransactionStatus.Confirmed
+            ? "text-transparent bg-clip-text brand-gradient-2"
```

Comment:
figma is using `green-800`
```suggestion
            ? "text-transparent bg-clip-text brand-gradient-2"
```